rama-pac/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub mod connector;
+pub mod pac_fetcher;
```

Comment:
no need to make all modules public if you only use 1 or 2 types it's better to keep the public module structure flat.